project_euler/problem_25/sol1.py

```diff
@@ -25,8 +25,13 @@
 """
 
 
-def fibonacci(n):
-    if n == 1 or type(n) is not int:
+def fibonacci(n: int) -> int:
+    """
+    Computes the Fibonacci number for input n by iterating through n numbers
+    and creating an array of ints using the Fibonacci formula.
+    Returns the nth element of the array.
+    """
```

Comment:
It will be great if you were to write doctests for all the functions as well

Reply:
okay I have added doctests for all the functions to.